### PR TITLE
XLSX export/download

### LINKED ISSUE #8: XLSX export/download
Implement a similar action like the current CSV export action, but use https://github.com/Neovici/nullxlsx/ to export/download XLSX.

### PR DESCRIPTION
**Pull request by programmer4web**
Added XLSX export/download

fixes #8

### CODE REVIEW

File: import-nullxlsx.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<script src="../nullxlsx/dist/nullxlsx.min.js"></script>
```
Comment (nomego) [inline import-nullxlsx.html]: Wouldn't it make more sense to have this in the nullxlsx repo?

File: cosmoz-omnitable.js [hunk 2]
```diff
@@ -860,6 +876,40 @@
 			}));
 		},
 
+		/**
+		 * Makes the data ready to be exported as XLSX.
+		 * @returns {Array} data Array of prepared rows.
+		 */
+		_prepareXlsxData: function () {
```
Comment (nomego) [inline cosmoz-omnitable.js]: The biggest advantage of saving as XLSX over CSV is the ability to describe the column types, which this doesn't seem to do.
Not having tested the code, I would assume that Excel would have similar difficulties as with CSV to figure out what is numbers, dates, money, etc without explicit column types, but I might be wrong?

Comment (programmer4web) [reply cosmoz-omnitable.js]: At the moment it doesn't handle data types as String is being used.
It seems that the library supports data types so I will try to export it with correct data type.

File: cosmoz-omnitable-column-date-behavior.html [hunk 3]
```diff
@@ -246,6 +246,11 @@
 			return moment(value).format(this.userFormat);
 		},
 
+		toXlsxValue: function () {
```
Comment (megheaiulian) [inline cosmoz-omnitable-column-date-behavior.html]: @programmer4web Revert change to `OmnitableDateBehavior` because `moment` will be removed in  #59.

**Pull request merged**